Apps/ValidationTests/Scripts/validation_native.js

```diff
@@ -6,7 +6,7 @@ var justOnce;
 var saveResult = true;
 var testWidth = 600;
 var testHeight = 400;
-
+var generateReferences = false;
```

Comment:
I guess this is deviating even more from the Babylon.js version now? It would be good to figure out a way such that more code is shared between Babylon.js and this code or at least make it easy to update.

Reply:
Yes, the code is deviating even more. Same with the datas. For this rather small script, I think trying to make it more shareable with bjs will make it more complex and more difficult to update. The script in bjs repo is not part of any npm, it does things differently (using canvas Vs using TestUtils). I don't see how benefical the time spent to make it more generic will be. I'm more interested in making the datas completely shareable between js and native. Test script does not change as much as the datas. And for now, I know PGs are updated on the JS side and they are not on the native side.

Reply:
I'm good with that. It would be good to clean up this file. I did this a while back and didn't merge it because we wanted to share this code with the JS side.